cedar-wasm-example/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "cedar-wasm-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.ts",
+  "scripts": {
+    "test": "jest",
+    "start": "ts-node main.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
```

Comment:
I think for our other code we generally use Apache-2.0. Is there a reason to use MIT here?